load_dir.h

```diff
@@ -82,4 +82,17 @@ static inline void ptr_array_unique(struct ptr_array *array,
 	array->count = j;
 }
 
+static inline void ptr_array_clear(struct ptr_array *array){
+	struct dir_entry **ents;
+
+	ents = array->ptrs;
```

Comment:
In theory the container is type agnostic so it would be better to follow `ptr_array_unique`'s lead with:
`void **ptrs = array->ptrs;`

Reply:
fixed